Workbooks/Azure Monitor - Workspaces/Container Insights Usage/Container Insights Usage.workbook

```diff
@@ -0,0 +1,257 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Container Insights billable data usage\n\n\n#### This workbook provides you a summary & source for billable data collected by [Container Insights solution.](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/container-insights-overview)\n\nThe best way to understand how Container Insights ingest data in Log Analytics workspace is from this diagram<img src=\"https://microsoft-my.sharepoint-df.com/personal/saaror_microsoft_com/_layouts/15/onedrive.aspx?FolderCTID=0x0120001706329421A9C9488B31E57BE5044A3D&id=%2Fpersonal%2Fsaaror%5Fmicrosoft%5Fcom%2FDocuments%2FAttachments%2FCost%5Fbreakdown%2Ejpg&parent=%2Fpersonal%2Fsaaror%5Fmicrosoft%5Fcom%2FDocuments%2FAttachments\" \nalt=\"IMAGE ALT TEXT HERE\" width=\"240\" height=\"180\" border=\"10\" />\n\nIn this workbook you can understand your data by\n\n* You can view billable data ingested by **solution**.\n* You can view billable data ingested by **Container logs(application logs)**\n* You can view billable container logs data ingested segregated by **Kubernetes namespace**\n* You can view billable container logs data ingested segregated by **Cluster name**\n* You can view billable container log data ingested by **logsource entry**\n* You can view billable diagnostic data ingested by **diagnostic master node logs**\n\nYou can fine tune & control logging by turning off logging on the above mentioned vectors. [Learn more how to do that](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/container-insights-agent-config)\n\nYou can control your master nodes logs by updating the diagnostic settings. [Learn more here](https://docs.microsoft.com/en-us/azure/aks/view-master-logs)\n\n\n</br>"
+      },
+      "name": "text - 2",
+      "styleSettings": {
+        "progressStyle": "loader",
+        "showBorder": true
+      }
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "cbafb196-4c70-4f43-b189-1de7620db19e",
+            "version": "KqlParameterItem/1.0",
+            "name": "Time_range",
+            "type": 4,
+            "description": "Select time-range for data selection",
+            "isRequired": true,
+            "value": {
+              "durationMs": 7776000000
```

Comment:
you have your default set to 90 days, which is going make the queries in the workbook worst case.  see [smallest possible time ranges by default](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/DataSources/LogsBestPracticesAndHints.md#use-the-smallest-possible-time-ranges-by-default)

Reply:
Fixed